src/Platforms/tests/OpenMP/test_class_member.cpp

```diff
@@ -15,8 +15,8 @@
 #include <memory>
 #include <vector>
 #include <iostream>
-#include "omp.h"
-#include "OMPallocator.hpp"
+#include <omp.h>
+#include <OpenMP/OMPallocator.hpp>
```

Comment:
Quotes again?

Reply:
`omp.h` is from compiler or OS. So use angle bracket.
`OpenMP/OMPallocator.hpp` is from QMCPACK, so should be quotes.
I fixed this place in this PR but need to fix other files in the repo using a script in a separate PR.